spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -32,25 +32,30 @@
 
   def then_i_see_citizen_and_business_results
     then_i_should_see_the_results_page
+    and_i_should_see_citizen_actions_are_grouped
     and_i_should_see_the_citizens_action_header
     and_i_should_see_the_business_action_header
     and_i_should_see_a_pet_action
     and_i_should_see_a_tourism_action
+    and_the_tourism_link_should_have_tracking_analyitics
   end
 
   def then_i_see_business_results_only
     then_i_should_see_the_results_page
     expect(page).to_not have_content I18n.t!("brexit_checker.results.audiences.citizen.heading")
```

Comment:
I'm aware this isn't your change, but it seems like quite a small tidy-up task.

I would pull this out into something like the scenario above, where you have a method for: `and_i_should_see_the_citizens_action_header`. 

So: `and_i_should_not_see_the_citizens_action_header`

Just to keep the tests consistent in not expecting anything inside each scenario.

Reply:
done!